samples/hello_world/prj.conf

```diff
@@ -1 +1,30 @@
 # nothing here
+CONFIG_NETWORKING=y
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_L2_IEEE802154=y
+CONFIG_IEEE802154_CC13XX_CC26XX=n
+CONFIG_IEEE802154_CC13XX_CC26XX_SUB_GHZ=y
+CONFIG_NET_CONFIG_IEEE802154_DEV_NAME="IEEE802154_1"
+CONFIG_IEEE802154=y
+CONFIG_LOG=y
+CONFIG_IEEE802154_DRIVER_LOG_LEVEL_DBG=y
+CONFIG_NET_L2_IEEE802154_SHELL=y
+CONFIG_NET_SHELL=y
+CONFIG_NET_CONFIG_IEEE802154_PAN_ID=0xabcd
+CONFIG_NET_CONFIG_IEEE802154_CHANNEL=1
+
+CONFIG_NO_OPTIMIZATIONS=y
+CONFIG_BUILD_OUTPUT_HEX=y
+
+# use this to measure stack usage
+CONFIG_KERNEL_SHELL=y
+CONFIG_INIT_STACKS=y
+
+# These were both 1024 by default, and that was overflown fairly quickly
+# Need to measure stack sizes for all threads that are created
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+# Power management
+CONFIG_SYS_POWER_MANAGEMENT=y
+CONFIG_SYS_POWER_SLEEP_STATES=y
```

Comment:
Hmm, I do not think we should add any networking config to hello_world application.